src/Messaging/Messages.ts

```diff
@@ -12,6 +12,11 @@ export interface LightBlock {
   readonly height: number
 }
 
+export interface HashTxId {
```

Comment:
`Hash` feels too generic here to me. Without context it could get confusing. Maybe rename to `IPFSDirectoryHashTransactionId`?

Reply:
lol of course

Reply:
You're already using `ipfsHashTxId` for the exchange name, so maybe just go with that in pascal case?

Reply:
yeah that's more consistent